docs/conversion_examples_gallery/recording/neuroscope.rst

```diff
@@ -0,0 +1,31 @@
+Neuroscope data conversion
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Convert Neuroscope data to NWB using :py:class:`~nwb_conversion_tools.datainterfaces.NeuroscopeRecordingInterface`.
+
+.. code-block:: python
+
+    >>> from datetime import datetime
+    >>> from dateutil import tz
+    >>> from pathlib import Path
+    >>> from nwb_conversion_tools import NeuroscopeRecordingInterface
+    >>> 
+    >>> # For Neuroscope we need to pass the location of the `.dat` file 
+    >>> file_path = f"{ECEPHY_DATA_PATH}/neuroscope/test1/test1.dat"
+    >>> # Change the file_path to the location in your system
+    >>> interface = NeuroscopeRecordingInterface(file_path=file_path)
+    >>> 
+    >>> # Extract what metadata we can from the source files
+    >>> metadata = interface.get_metadata()
+    >>> # session_start_time is required for conversion. If it cannot be inferred 
+    >>> # automatically from the source files you must supply one.
+    >>> session_start_time = datetime(2020, 1, 1, 12, 30, 0, tzinfo=tz.gettz("US/Pacific"))
+    >>> metadata["NWBFile"] = dict(session_start_time=session_start_time)
```

Comment:
```suggestion
    >>> metadata["NWBFile"].update(session_start_time=session_start_time)
```

Reply:
This will fail because in this example the metadata does not have the key `NWBFile` already.

Reply:
ah